### PR TITLE
Chinese translation completed!

### PR DESCRIPTION
**Pull request by btwise**
Modify and add Chinese translations of all files, add all unadded object translations in miscController.strings!

### CODE REVIEW

File: Chinese, Simplified Localization (zh-Hans)/devicePropertiesController.strings [hunk 1]
```diff
@@ -1,41 +1,34 @@
...
+"dpG-4m-Tya.headerCell.title" = "类型";
+
 /* Class = "NSTableColumn"; headerCell.title = "Devices"; ObjectID = "9kg-WA-Eir"; */
 "9kg-WA-Eir.headerCell.title" = "设备";
 
 /* Class = "NSTextFieldCell"; title = "Text Cell"; ObjectID = "CrE-0l-DLY"; */
 "CrE-0l-DLY.title" = "Text Cell";
 
-/* Class = "NSTableColumn"; headerCell.title = "Type"; ObjectID = "dpG-4m-Tya"; */
-"dpG-4m-Tya.headerCell.title" = "Type";
```
Comment (mackie100) [inline Chinese, Simplified Localization (zh-Hans)/devicePropertiesController.strings]: This is a header cell title. Required!

File: Chinese, Simplified Localization (zh-Hans)/devicePropertiesController.strings [hunk 2]
```diff
@@ -1,41 +1,34 @@
...
-
-/* Class = "NSTextFieldCell"; title = "Text Cell"; ObjectID = "eia-C9-4Iw"; */
-"eia-C9-4Iw.title" = "Text Cell";
-
 /* Class = "NSTableColumn"; headerCell.title = "Key*"; ObjectID = "F6A-yy-QfQ"; */
 "F6A-yy-QfQ.headerCell.title" = "键值*";
 
-/* Class = "NSMenuItem"; title = "NUMBER"; ObjectID = "lI7-XR-ash"; */
-"lI7-XR-ash.title" = "NUMBER";
-
```
Comment (mackie100) [inline Chinese, Simplified Localization (zh-Hans)/devicePropertiesController.strings]: DATA, STRING, NUMBER are menu items. Required!

File: Chinese, Simplified Localization (zh-Hans)/devicePropertiesController.strings [hunk 3]
```diff
@@ -1,41 +1,34 @@
...
 /* Class = "NSTextFieldCell"; title = "Text Cell"; ObjectID = "M7c-ev-b9D"; */
 "M7c-ev-b9D.title" = "Text Cell";
 
-/* Class = "NSTabViewItem"; label = "Block"; ObjectID = "n3V-ht-3yU"; */
-"n3V-ht-3yU.label" = "阻止";
-
```
Comment (mackie100) [inline Chinese, Simplified Localization (zh-Hans)/devicePropertiesController.strings]: item required for tabview like a Add, Block, Patch

File: Chinese, Simplified Localization (zh-Hans)/devicePropertiesController.strings [hunk 4]
```diff
@@ -1,41 +1,34 @@
...
-"ogK-XX-Zb9.ibShadowedToolTip" = "1. Add\nType: plist dict\nDescription: Sets device properties from a map (plist dict) of deivce paths to a map (plist dict) of variable names and their values in plist metadata format. Device paths must be provided in canonic string format (e.g. PciRoot(0x0)/Pci(0x1,0x0)/Pci(0x0,0x0)). Properties will only be set if not present and not blocked.\nNote: Currently properties may only be (formerly) added by the original driver, so unless a separate driver was installed, there is no reason to block the variables.";
+/* Class = "NSTextFieldCell"; title = "Text Cell"; ObjectID = "eia-C9-4Iw"; */
+"eia-C9-4Iw.title" = "Text Cell";
 
-/* Class = "NSMenuItem"; title = "DATA"; ObjectID = "POL-aB-DNF"; */
-"POL-aB-DNF.title" = "DATA";
```
Comment (mackie100) [inline Chinese, Simplified Localization (zh-Hans)/devicePropertiesController.strings]: required

File: Chinese, Simplified Localization (zh-Hans)/devicePropertiesController.strings [hunk 5]
```diff
@@ -1,41 +1,34 @@
-/* Class = "NSView"; ibShadowedToolTip = "2. Block\nType: plist dict\nDescription: Removes device properties from a map (plist dict) of deivce paths to an array (plist array) of variable names in plist string format."; ObjectID = "1eh-jH-TX2"; */
-"1eh-jH-TX2.ibShadowedToolTip" = "2. Block\nType: plist dict\nDescription: Removes device properties from a map (plist dict) of deivce paths to an array (plist array) of variable names in plist string format.";
```
Comment (mackie100) [inline Chinese, Simplified Localization (zh-Hans)/devicePropertiesController.strings]: ???

File: Chinese, Simplified Localization (zh-Hans)/devicePropertiesController.strings (hunk 1 above)
Comment (mackie100) [inline Chinese, Simplified Localization (zh-Hans)/devicePropertiesController.strings]: required

Comment (btwise) [reply Chinese, Simplified Localization (zh-Hans)/devicePropertiesController.strings]: "dpG-4m-Tya.headerCell.title" = "类型";

File: Chinese, Simplified Localization (zh-Hans)/devicePropertiesController.strings (hunk 5 above)
Comment (btwise) [reply Chinese, Simplified Localization (zh-Hans)/devicePropertiesController.strings]: OK, I resubmit the changes!

File: Chinese, Simplified Localization (zh-Hans)/devicePropertiesController.strings (hunk 4 above)
Comment (btwise) [reply Chinese, Simplified Localization (zh-Hans)/devicePropertiesController.strings]: OK, I resubmit the changes!

File: Chinese, Simplified Localization (zh-Hans)/devicePropertiesController.strings (hunk 3 above)
Comment (btwise) [reply Chinese, Simplified Localization (zh-Hans)/devicePropertiesController.strings]: OK, I resubmit the changes!

File: Chinese, Simplified Localization (zh-Hans)/devicePropertiesController.strings (hunk 2 above)
Comment (btwise) [reply Chinese, Simplified Localization (zh-Hans)/devicePropertiesController.strings]: OK, I resubmit the changes!

File: Chinese, Simplified Localization (zh-Hans)/devicePropertiesController.strings (hunk 1 above)
Comment (btwise) [reply Chinese, Simplified Localization (zh-Hans)/devicePropertiesController.strings]: OK, I resubmit the changes!

File: Chinese, Simplified Localization (zh-Hans)/devicePropertiesController.strings [hunk 6]
```diff
@@ -45,7 +38,3 @@
 
 /* Class = "NSTextFieldCell"; title = "Text Cell"; ObjectID = "tAW-lD-Zil"; */
 "tAW-lD-Zil.title" = "Text Cell";
-
-/* Class = "NSMenuItem"; title = "STRING"; ObjectID = "ycS-j8-cj1"; */
-"ycS-j8-cj1.title" = "STRING";
-
```
Comment (mackie100) [inline Chinese, Simplified Localization (zh-Hans)/devicePropertiesController.strings]: required

**Comment by btwise** [comment]
I have re-edited and submitted it. Can you see it?

**Comment by mackie100** [comment]
> I have re-edited and submitted it. Can you see it?

Ok...

**Comment by btwise** [comment]
Can you add a list of frequently renamed patches and kernel patch list selection?
We also have a variety of ACPI patches for the OC bootloader, you can also choose to download!

**Pull request merged**